test/suites/optional/olm.robot

```diff
@@ -0,0 +1,136 @@
+*** Settings ***
+Documentation       Operator Lifecycle Manager on MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${CATALOG_SOURCE}           ./assets/olm/catalog-source.yaml
+${SUB_CERT_MANAGER}         ./assets/olm/subscription-cert-manager.yaml
+${OPERATORS_NAMESPACE}      openshift-operators
+
+
+*** Test Cases ***
+Deploy CertManager From OperatorHubIO
+    [Documentation]    Deploy CertManager from OperatorHub Catalog.
+    [Setup]    OLM Should Be Ready
+
+    Create OperatorHub CatalogSource
```

Comment:
We can only put `Create OperatorHub CatalogSource` and `Create CertManager Subscription` because AFAIK `Run Keywords` does not support variable assignment. Shall we proceed that way?

Reply:
I see it's been done already.